Go/Go.sublime-syntax

```diff
@@ -916,6 +916,8 @@ contexts:
       pop: true
     - match: '{{predeclared_type}}'
       scope: support.type.go
+    - match: (?=;)
+      pop: 1
```

Comment:
If improving support for incomplete statements is the goal, it may also make sense to pop on stray closing brackets and keywords, which are not valid in expressions, such as `case`, `if`, ...

Reply:
maybe let's leave that for a future PR then, it sounds quite a bit more involved :) if needed, I can drop the commit which fixed the `case` statement pop at semi-colon from this PR.

Reply:
It is not an error to have it there, even if more cases may be relevant. It's fine to address them later.